### PR TITLE
ci: add release workflow

### PR DESCRIPTION
**Pull request by alexanderkranga**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+jobs:
+  release:
+    name: Release to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (daniftodi) [inline .github/workflows/release.yml]: why do you need `fetch-depth`?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.sass-cache/
+/connect.lock
+/coverage
+/libpeerconnection.log
+testem.log
+/typings
+
+# System files
+.DS_Store
+Thumbs.db
```
Comment (daniftodi) [inline .gitignore:40]: should we add the `lib/` directory here as well?

**Review by daniftodi — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (alexanderkranga) [reply .gitignore:40]: yes, it is required

File: .github/workflows/release.yml (hunk 1 above)
Comment (alexanderkranga) [reply .github/workflows/release.yml]: will remove this

**Pull request merged**